src/test/java/core/basesyntax/service/fileoperation/impl/CsvFileReaderImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.service.fileoperation.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.fileoperation.CsvFileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvFileReaderImplTest {
+    private static final Path TEST_FILE_PATH = Path.of("src/test/resources.csv");
+    private static final Path INVALID_TEST_FILE_PATH = Path.of("src/test/invalidResources.csv");
+    private static CsvFileReader reader;
+    private static List<String> validTestData;
+    private static List<String> emptyTestData;
+
+    private void createTestFile(String stringForWrite) {
+        try {
+            Files.write(TEST_FILE_PATH, stringForWrite.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Data is not written. ");
```

Comment:
not informative exception message 
```suggestion
            throw new RuntimeException("Cant write data to file " + TEST_FILE_PATH, e);
```